plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardPage.java

```diff
@@ -106,10 +108,26 @@ protected void doCreateControls(Composite parent, DataBindingContext dbc) {
 			.to(signupUrlObservable)
 			.converting(new IsNotEmptyString2BooleanConverter())
 			.in(dbc);
-		
+
+		// userdoc link (JBIDE-20401)
+		Link userdocLink = new Link(parent, SWT.WRAP);
+		userdocLink.setText("Note: OpenShift v3 behaves differently than v2. See <a>documentation here</a>.");
```

Comment:
shouldn't we be just updating the signuplink text in case user choose OSv3 rather than adding a new additional line/link ?